MAUI/Calendar/date-navigations.md

```diff
@@ -0,0 +1,172 @@
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month"
+                        AllowViewNavigation="True">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.AllowViewNavigation = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Programmatically change to adjacent dates
+The next and previous view can be navigated by swiping the Calendar control from right to left and left to right. In the `SfCalendar`, view can be changed programmatically by using the [Forward](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_Forward) and [Backward](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_Backward) methods.
+
+### Forward
+The `Forward` navigation allows you to view the next immediate date of the calendar based on the CalendarViews.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition />
+            <RowDefinition Height="30"/>
+        </Grid.RowDefinitions>
+        <Calendar:SfCalendar x:Name="Calendar"
+                             View="Month" />
+        <Button x:Name="button" Grid.Row="1" Text="Forward"
+                Clicked="button_Clicked" />
+</Grid>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+private void button_Clicked(object sender, EventArgs e)
+{
+  this.Calendar.Forward();
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Backward
+The `Backward` navigation allows you to view the immediate previous date of the calendar based on the CalendarViews.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition />
+            <RowDefinition Height="30"/>
+        </Grid.RowDefinitions>
+        <Calendar:SfCalendar x:Name="Calendar"
+                             View="Month" />
+        <Button x:Name="button" Grid.Row="1" Text="Backward"
+                Clicked="button_Clicked" />
+</Grid>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+private void button_Clicked(object sender, EventArgs e)
+{
+  this.Calendar.Backward();
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Navigation direction
+Views can be navigated by using the `Navigation direction` property either `Vertical` or `Horizontal` directions by setting the `NavigationDirection` property of `Calendar`. The default [NavigationDirection](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_NavigationDirection) is `Vertical`.
+
+The following code shows the Navigation direction in `Horizontal`,
```

Comment:
The following code shows the Navigation in the `Horizontal` direction.

Reply:
Content modified